setup.py

```diff
@@ -29,7 +29,7 @@
 with open(path.join(here, 'requirements.txt')) as requirements_file:
     # Parse requirements.txt, ignoring any commented-out lines.
     requirements = [line for line in requirements_file.read().splitlines()
-                    if not (line.startswith('#') or line.startswith('git+'))]
+                    if not line.startswith('#')]
```

Comment:
This was also bad but it shows up in some of our other repositories. I guess I should treat this as something to be removed rather than reproduced.